core/server/common/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -64,12 +64,19 @@
       + "OPTIONS can be a list of command line options. Each option has the"
       + " format \"-<optionName> [optionValue]\"\n";
 
+  private static final Options OPTIONS = new Options();
+
   private static final Map<ValidationTask, String> TASKS = new HashMap<>();
 
   private static final String ALLUXIO_MASTER_CLASS = "alluxio.master.AlluxioMaster";
   private static final String ALLUXIO_WORKER_CLASS = "alluxio.worker.AlluxioWorker";
   private static final String ALLUXIO_PROXY_CLASS = "alluxio.proxy.AlluxioProxy";
 
+  // HDFS configuration validations
+  private static final ValidationTask HDFS_VALIDATION_TASK = registerTask(
+      "hdfs",
```

Comment:
(nit) Can you give it a little bit more specific name? Something like `ufs.hdfs.config.consistency`?

Reply:
Done.

Reply:
Done.